app/scripts/actions/index.js

```diff
@@ -398,6 +398,15 @@ export const clearGranulesSearch = () => ({ type: types.CLEAR_GRANULES_SEARCH })
 export const filterGranules = (param) => ({ type: types.FILTER_GRANULES, param: param });
 export const clearGranulesFilter = (paramKey) => ({ type: types.CLEAR_GRANULES_FILTER, paramKey: paramKey });
 
+export const getGranuleCSV = (options) => ({
+  [CALL_API]: {
+    type: types.GRANULE_CSV,
+    method: 'GET',
+    url: url.resolve(root, 'granule-csv'),
+    qs: Object.assign({}, options)
```

Comment:
You don't need query state at all, right?

Reply:
nope, will remove